Zend/tests/name_collision_08.phpt

```diff
@@ -13,4 +13,4 @@ namespace Bazzle {
 }
 ?>
 --EXPECTF--
-Fatal error: Cannot declare function Bazzle\bar because the name is already in use in %s on line %d
+Fatal error: Cannot redeclare Bazzle\bar() (previously declared as local import) in %s on line %d
```

Comment:
The function keyword disappeared from the error message. I just wanted to point that it's inconsistent with other messages where the type of construct is mentioned.

I don't know whether this is intended or overlooked.

```suggestion
Fatal error: Cannot redeclare function Bazzle\bar() (previously declared as local import) in %s on line %d
```

Reply:
It's intended, I adjusted it to match the existing one for actual redeclarations. https://3v4l.org/P83Jp

Reply:
I agree that it's better though to adjust the other case instead.